Crash on opening gyroscope configuration menu

**Actual Behaviour**

crash when configuration menu is opened in gyroscope

**Expected Behaviour**

crash should not happen

**Steps to reproduce it**

Open gyroscope configuration

**LogCat for the issue**

 Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.support.v7.preference.EditTextPreference.getText()' on a null object reference
        at io.pslab.fragment.GyroscopeSettingsFragment.onResume(GyroscopeSettingsFragment.java:51)
        at android.support.v4.app.Fragment.performResume(Fragment.java:2498)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1501)
        at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at android.support.v4.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3269)
        at android.support.v4.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:3241)
        at android.support.v4.app.FragmentController.dispatchResume(FragmentController.java:223)
        at android.support.v4.app.FragmentActivity.onResumeFragments(FragmentActivity.java:538)
        at android.support.v4.app.FragmentActivity.onPostResume(FragmentActivity.java:527)
        at android.support.v7.app.AppCompatActivity.onPostResume(AppCompatActivity.java:172)
        at android.app.Activity.performResume(Activity.java:7583)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4007)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4047) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1955) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7073) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:974) 

**Screenshots of the issue**

N/A

**Would you like to work on the issue?**

Yes

--- a/app/src/main/res/xml/gyro_scope_settings.xml
+++ b/app/src/main/res/xml/gyro_scope_settings.xml
@@ -8,7 +8,7 @@
         android:dialogMessage="@string/time_string_baro"
         android:dialogTitle="@string/update_period"
         android:inputType="numberDecimal"
-        android:key="@string/setting_lux_update"
+        android:key="@string/setting_gyro_update"
         android:summary="@string/lux_summary"
         android:title="@string/update_period"
         app:iconSpaceReserved="false" />
@@ -18,7 +18,7 @@
         android:dialogMessage="@string/dialogue_msg_gyro"
         android:dialogTitle="@string/high_limit"
         android:inputType="numberDecimal"
-        android:key="@string/high_limit_lux"
+        android:key="@string/high_limit_gyro"
         android:summary="@string/gyro_default_20"
         android:title="@string/high_limit"
         app:iconSpaceReserved="false" />
@@ -29,7 +29,7 @@
         android:dialogMessage="@string/set_gain_sensor"
         android:dialogTitle="@string/sensor_gain"
         android:inputType="numberDecimal"
-        android:key="@string/setting_lux_sensor_gain"
+        android:key="@string/setting_gyro_sensor_gain"
         android:summary="@string/default_value_lux"
         android:title="@string/sensor_gain"
         app:iconSpaceReserved="false" />

